payments-core/src/main/java/com/stripe/android/model/parsers/DeferredSetupIntentJsonParser.kt

```diff
@@ -4,14 +4,13 @@ import androidx.annotation.RestrictTo
 import com.stripe.android.core.model.StripeJsonUtils.optString
 import com.stripe.android.core.model.parsers.ModelJsonParser
 import com.stripe.android.core.model.parsers.ModelJsonParser.Companion.jsonArrayToList
-import com.stripe.android.model.DeferredIntentParams
 import com.stripe.android.model.SetupIntent
+import com.stripe.android.model.StripeIntent
 import org.json.JSONObject
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 class DeferredSetupIntentJsonParser(
     private val elementsSessionId: String?,
-    private val params: DeferredIntentParams,
```

Comment:
We don’t actually need this here, because (at least in my understanding) we’ll always use `OffSession` for setup intents.

Reply:
We allow the merchant to specify `onSession` too, though.  We might have some future logic that does different things depending on the merchant's `setupFutureUsage` value so I'm not sure we want to ignore the merchant value here.